packages/nextjs/src/utils/serverIntegrations.ts

```diff
@@ -0,0 +1,55 @@
+import { RewriteFrames } from '@sentry/integrations';
+import { Integration } from '@sentry/types';
+
+const SOURCEMAP_FILENAME_REGEX = /^.*\/.next\//;
```

Comment:
```suggestion
const SOURCEMAP_FILENAME_REGEX = /^.*\/\.next\//;
```

We're looking for a literal `.`, not any char.

Reply:
good one